e2e/tests/admin/whats-new.constants.ts

```diff
@@ -0,0 +1,2 @@
+export const WHATS_NEW_WAIT_TIMEOUT = 5000;
```

Comment:
have a feeling we can probably ditch this file by moving timeouts to page objects what's new aread, and date as value to test